ModelCatalogueCorePlugin/src/groovy/org/modelcatalogue/core/publishing/changelog/AbstractChangeLogGenerator.groovy

```diff
@@ -271,6 +271,18 @@ abstract class AbstractChangeLogGenerator {
         }.items
     }
 
+    protected List<Change> getChangesAfterDate(CatalogueElement element, Date startDate, ChangeType... types) {
+        auditService.getChanges(element, sort: 'dateCreated', order: 'asc'){
+            ne 'undone', true
+            isNull 'parentId'
+            gt 'dateCreated', startDate
+
+            if (types) {
+                inList 'type', types.toList()
+            }
+        }.items
+    }
+
```

Comment:
do you keep in mind the correct order of the columns to fit to existing indexes?